load_tests/load_test.py

```diff
@@ -32,6 +35,13 @@
     },
 ]
 
+PLUGIN_NAME_MAPS = {
+    "kinesis": "kinesis_streams",
+    "firehose": "kinesis_firehose",
+    "s3": "s3",
+    "cloudwatch": "cloudwatch",
```

Comment:
should be `cloudwatch_logs` as the value I think...

Reply:
great. will change

Reply:
changed